test/contrib/sge_test.py

```diff
@@ -21,9 +21,10 @@
 from glob import glob
 import unittest
 import logging
+from mock import patch
 
 import luigi
-from luigi.contrib.sge import SGEJobTask, _parse_qstat_state, _clean_task_id
```

Comment:
Oh, I didn't know it was optional to leave out the python version. Cool.

Anyway the command looks sane. You might want to see the COTNRIBUTING file too though.